superset/alfred/utils.py

```diff
@@ -0,0 +1,350 @@
+from marshmallow.validate import ValidationError
+import pandas as pd
+import traceback
+import jwt
+import os
+from superset.utils import core as utils
+from superset.exceptions import SupersetException
+
+
+from trino.dbapi import connect
+from trino.auth import JWTAuthentication
+from flask import request, make_response, current_app, current_app
+
+from hogwarts.auth.vault.vault_client import VaultClient
+
+# This will by default restrict this function to users with ALPR.
+# You can customize this by providing an azure group ID to the wrapper.
+
+import datetime
+
+from trino.client import NamedRowTuple
+
+from alfred_client import AlfredService
+from alfred_client.instance import InstanceUtil
+from alfred_client import RetentionInstance
+
+import logging
+import os.path
+from datetime import datetime
+from datetime import timedelta
+from typing import Optional, Union
+import json
+
+from alfred_client.instance import InstanceUtil
+from alfred_client.instance.AbstractInstance import AbstractInstance
+
+# Determine if we are in JupyHub Hogwarts environment by the presence of the hogwarts library.
+
+import jwt
+
+logger = logging.getLogger(__name__)
+
+# limit to 100 emls retained at a time
+limit = 100
```

Comment:
This doesn't match up with our 30 selected EML IDs allowed in the right-click menu. Just checking if this is intentional?

Reply:
this was initially the limit in the fission function, we imposed the 30 limit in superset. I can raise the superset limit to 100 now

Reply:
Hopefully it still performs with 100!